axelor-production/src/main/java/com/axelor/apps/production/service/SaleOrderLineBomServiceImpl.java

```diff
@@ -98,4 +104,20 @@ public List<SaleOrderLine> createSaleOrderLinesFromBom(
         .sorted(Comparator.comparingInt(SaleOrderLine::getSequence))
         .collect(Collectors.toList());
   }
+
+  @Override
+  public boolean ShowQtyProduced(SaleOrderLine saleOrderLine) {
+    SaleOrder saleOrder = saleOrderLine.getSaleOrder();
+
+    ManufOrder manufOrder =
+        manufOrderRepository
+            .all()
+            .filter("self.product.id=:productId and :saleOrderLineId =self.saleOrderLine.id")
+            .bind("saleOrderLineId", saleOrderLine.getId())
+            .bind("productId", saleOrderLine.getProduct().getId())
+            .fetchOne();
+
+    return saleOrder.getStatusSelect() >= SaleOrderRepository.STATUS_ORDER_CONFIRMED
+        && manufOrder != null;
+  }
```

Comment:
As said in the SaleOrderLine.xml comment, this code should be in a service SaleOrderViewProductionServiceImpl. 

Also there is a mistake in the method name, first letter should not be in uppercase.